spec/request/disability_claims_spec.rb

```diff
@@ -37,4 +37,14 @@
       expect(response).to have_http_status(:internal_server_error)
     end
   end
+
+  context 'with an outage' do
+    before do
+      EVSS::ClaimsService.breakers_service.begin_forced_outage!
```

Comment:
Can this be moved to `spec/lib/evss/base_service_spec.rb` and use the BaseService methods to test the breakers functionality? Does breakers handle both GET and POST? If so, adding a test for both those methods on BaseService would be 🅰️ ➕